optimize/self-managed/optimize-deployment/configuration/system-configuration-platform-7.md

```diff
@@ -69,6 +69,7 @@ REST API endpoint locations, timeouts, etc.
 | import.importIndexStorageIntervalInSec                    | 10            | States how often the import index should be stored to Elasticsearch.                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
 | import.currentTimeBackoffMilliseconds                     | 300000        | This is the time interval the import backs off from the current tip of the time during the ongoing import cycle. This ensures that potentially missed concurrent writes in the engine are reread going back by the amount of this time interval.                                                                                                                                                                                                                                                                          |
 | import.identitySync.includeUserMetaData                   | true          | Whether to include metaData (firstName, lastName, email) when synchronizing users. If disabled only user IDs will be shown on user search and in collection permissions.                                                                                                                                                                                                                                                                                                                                                  |
+| import.identitySync.collectionRoleCleanupEnabled          | true          | Whether collection role cleanup should be performed                                                                                                                                                                                                                                                                                                                                                                                                                                                                       |
```

Comment:
I wouldn't expect users to know what the `collection role cleanup` does, could add a bit more explanation:
```suggestion
| import.identitySync.collectionRoleCleanupEnabled          | true          | Whether collection role cleanup should be performed. If disabled, users that no longer exist in the identity provider will not be automatically removed from collection permissions                                                                                                                                                                                                                                                                                                                                                                                                                                     |
```
though that then probably will lead people to assume they need this to be `true`. Up to you 🤷